cluster/manifests/appdynamics/daemonset.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: monitor
+    application: appdynamics-machineagent
+    version: v1
+  name: appdynamics-machineagent
+  namespace: kube-system
+spec:
+  template:
+    metadata:
```

Comment:
Here the update strategy and match selector should also be defined. See for an example: https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/logging-agent/daemonset.yaml#L12-L15